modules/test-tools/src/main/kotlin/jp/co/soramitsu/iroha2/testengine/IrohaRunnerExtension.kt

```diff
@@ -175,6 +175,7 @@ class IrohaRunnerExtension : InvocationInterceptor, BeforeEachCallback {
             keyPairs.add(generateKeyPair())
             portsList.add(findFreePorts(3)) // P2P + API + TELEMETRY
         }
+        val genesisKeyPair = keyPairs.first()
```

Comment:
```suggestion
        val genesisKeyPair = generateKeyPair()
```

how about this? we shouldn't be using the same keys for nodes and genesis